### PR TITLE
Update: Data publishing frequency

### PR DESCRIPTION
**Pull request by kmanaseryan**
Related issue: 
https://app.zenhub.com/workspaces/datagov-multi-tenant-ckan-5cf12b601437ff1e943f0aa2/issues/gsa/datagov-ckan-multi/525

## Screenshots
image

image

image

## Demo
https://www.loom.com/share/f215dc5293c8463a8913e8976a375dfb

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -e git+https://github.com/GSA/ckan.git@inventory#egg=ckan
 -e git+https://github.com/ckan/ckanext-harvest.git@13dbb1eea428e6b274f98c36bf29d42e62a70af7#egg=ckanext-harvest
 -e git+https://github.com/GSA/ckanext-datajson.git@master#egg=ckanext-datajson
--e git+https://github.com/GSA/USMetadata.git@master#egg=ckanext-usmetadata
+-e git+https://github.com/GSA/USMetadata.git@30e09a8752e795d8522f30233069e72d8242311a#egg=ckanext_usmetadata
```
Comment (anuveyatsu) [inline requirements.txt:4]: @kmanaseryan I believe here we should point to master branch again.

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
@kmanaseryan done the review :+1: only comment is about `requirements.txt` as I think it shouldn't be changed and `USMetadata` extension should be used from master branch but not a commit.

File: requirements.txt (hunk 1 above)
Comment (kmanaseryan) [reply requirements.txt:4]: @anuveyatsu agree!

**Comment by kmanaseryan** [comment]
@anuveyatsu updated `requirements.txt` 👍

**Review by anuveyatsu — APPROVED** [review]

**Pull request merged**